specifyweb/frontend/js_src/lib/components/Merging/CompareSubView.tsx

```diff
@@ -147,6 +147,10 @@ function MergeDialog({
     ...[mergedRecords, ...children].map((children) => children.length)
   );
 
+  const childrenLength = children.filter(
+    (child) => child && child.length > 1
+  ).length;
```

Comment:
shouldn't this be the following instead?

```tsx
  const childrenLength = children.filter(
    (child) => child && child.length > 0
  ).length;
```
and then:
```tsx
childrenLength > 1 &
```
(notice 1 instead of 0)

Reply:
yes it should. my bad